### ISSUE TITLE
Need to train Discriminator adversarially

### ISSUE THREAD

**Issue by liujiahao11** [YeongHyeon/GANomaly-PyTorch#2]
In your code, D was jointly trained based on sum of 3 loss function. As a variant of GAN, you should train G+E and D based on opposite loss function.
